contrib/src/templates/tera_templates.rs

```diff
@@ -8,7 +8,14 @@ use super::{TemplateInfo, TEMPLATE_DIR};
 lazy_static! {
     static ref TERA: Result<tera::Tera, String> = {
         let path: PathBuf = [&*TEMPLATE_DIR, "**", "*.tera"].iter().collect();
-        tera::Tera::new(path.to_str().unwrap()).map_err(|e| format!("{:?}", e))
+        let escape_ext = [".html.tera", ".htm.tera", ".xml.tera",
```

Comment:
I'd normally consider explicit naming more important to readability than vertical space; but I don't think this is worth re-litigating. I'll resubmit with your original edit.  Thanks for the review!